portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsNumericField.testcase

```diff
@@ -143,13 +143,58 @@ definition {
 	}
 
 	@description = "Verify that the "Is Less Than" option can be used to validate a Numeric Field"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test CanBeValidatedUsingIsLessThan {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-132854 CanBeValidatedUsingIsLessThan pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+		
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Numeric");
+		
+		Form.gotoAdvancedTab();
+
+		FormFields.enableCheckbox(fieldName = "Validation");
+
+		FormFields.editValidationTargetArgument(
```

Comment:
@Jamersonze, can you please rename this macro just to `editValidation`?

Thanks.

Reply:
I cant. This would lead to a conflict with a pre-existent macro called "editValidation". Besides, this macro works better because the path it uses for the select field has "contains" that in the case tags only the condition i input.